network-protection/network-protection-impl/src/main/res/layout/item_exclusion_list_app.xml

```diff
@@ -20,4 +20,5 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_marginVertical="@dimen/keyline_2"
+    app:leadingIconSize="medium"
```

Comment:
This is better to be set to `large` to be more consistent with AppTP UI.

Reply:
Picked up in review, thanks @karlenDimla !